components/script/dom/element.rs

```diff
@@ -1385,11 +1386,24 @@ impl Element {
     }
 
     pub(crate) fn summarize(&self) -> Vec<AttrInfo> {
-        self.attrs
+        let window = self.node.owner_window();
+        let computed_style = window.GetComputedStyle(self, None);
+        let display = computed_style.Display();
+
+        let mut attrs: Vec<AttrInfo> = self
+            .attrs
             .borrow()
             .iter()
             .map(|attr| attr.summarize())
-            .collect()
+            .collect();
+
+        attrs.push(AttrInfo {
```

Comment:
Can you determine the display type in [`Node::summarize`](https://github.com/servo/servo/blob/66583ce3c956ada4432aae88a7e3ca3c96d04a63/components/script/dom/node.rs#L1188) instead? It seems very hacky to append it to the attributes and then remove it later in the devtools server again. (And doing so causes the server to not report attributes called "display" anymore)

You can just add a new field to `NodeInfo` and store it there.

Reply:
I felt the same way. Will update as you suggest. Thanks.